controllers/workspace/http.go

```
@@ -32,87 +32,111 @@ import (
-var (
-	httpClient            *http.Client
+type HttpClientsHolder interface {
+	GetHttpClient(routingConfig *controller.RoutingConfig) *http.Client
+	GetHealthCheckHttpClient(routingConfig *controller.RoutingConfig) *http.Client
+}
+
+type DefaultHttpsClientHolder struct {
+	k8s    client.Client
+	logger logr.Logger
+
+	client                *http.Client
```

golangci-lint (2.12.2)

[error] field client is unused

(unused)